.werft/jobs/build/prepare.ts

```diff
@@ -62,21 +63,15 @@ function configureCoreDevAccess() {
     }
 }
 
-function decideHarvesterVMCreation(werft: Werft, config: JobConfig) {
-    if (config.withVM && !VM.vmExists({ name: config.previewEnvironment.destname })) {
-        prepareVM(werft, config)
-    } else {
-        werft.currentPhaseSpan.setAttribute("werft.harvester.created_vm", false)
-    }
-    werft.done(prepareSlices.BOOT_VM)
-}
-
```

Comment:
Any reason for this function to be deleted?

If you delete it, you're not closing the "BOOT_VM" slice anymore, while also not setting the `werft.harvester.created_vm` span attribute to false

Reply:
I am still working on it.

Reply:
Ah sorry 👍